scripts/local-dev.sh

```diff
@@ -249,14 +274,21 @@ start_local_services() {
     
     # Set environment variables
     export PROMETHEUS_URL="http://localhost:$THANOS_PORT"
+    export TEMPO_URL="https://localhost:$TEMPO_PORT"
+    export TEMPO_TENANT_ID="dev"
+    export TEMPO_TOKEN="$TOKEN"
     export LLAMA_STACK_URL="http://localhost:$LLAMASTACK_PORT/v1/openai/v1"
     export THANOS_TOKEN="$TOKEN"
     export METRICS_API_URL="http://localhost:$METRICS_API_PORT"
     export MCP_URL="http://localhost:$MCP_PORT"
-    export PROM_URL="http://localhost:$THANOS_PORT"
+    export PROM_URL="$PROMETHEUS_URL"
     # Set log level (override with PYTHON_LOG_LEVEL=DEBUG for more verbose logging)
     export PYTHON_LOG_LEVEL="${PYTHON_LOG_LEVEL:-INFO}"
 
+    # SSL verification settings for Tempo HTTPS
```

Comment:
Need this as Tempo is only available via `https`. When trying `http` the server rejects the request. This option disables ssl_verification check (for local testing)